Source/WebCore/platform/unix/SharedMemoryUnix.cpp

```diff
@@ -28,6 +28,10 @@
 #include "config.h"
 #include "SharedMemory.h"
 
+#if defined(__ANDROID__) || defined(ANDROID)
+#define HAVE_ASHAREDMEMORY 1
```

Comment:
Why do we need this? It's enabled always for android, why don't we just check `OS(ANDROID)` everywhere?

Reply:
To write less down below. I'll change it to `OS(ANDROID)`.